release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2716,7 +2716,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.15.9
+[id="ocp-4-15-10"]
+=== RHSA-2024:1887 - {product-title} 4.15.10 bug fix and security update
+
+Issued: 2024-04-25
+
+{product-title} release 4.15.10, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1887[RHSA-2024:1887] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1892[RHSA-2024:1892] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.10 --pullspecs
+----
+
+[id="ocp-4-15-10-bug-fixes"]
+==== Bug fixes
+
+* Previously, Clusters born before 4.7 will had signer keys for api-int endpoint updated unexpectedly when users upgraded to OpenShift Container Platform 4.15 due to the installer deleting the SecretTypeTLS and then recreating the secret with `kubernetes.io/tls` type.  With this release, the issue is resloved by the installer changing the secret type without deleting the secret. (link:https://issues.redhat.com/browse/OCPBUGS-31807[*OCPBUGS-31807*])
+
+* Previously,  when users imported image stream tags, ICSP (In-Circuit Serial Programming) and IDMS (Integrated Database Managing System) were not allowed to co-exist. OpenShift ignored any IDMS/ITMS created by the user and favored ICSP. With this release, they are allowed to co-exist so importing image stream tags will now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31469[*OCPBUGS-31469*])
```

Comment:
```suggestion
* Previously,  when users imported image stream tags, ICSP (In-Circuit Serial Programming) was not allowed to co-exist with IDMS (Integrated Database Managing System) and ImageTagMirrorSet (ITMS). OpenShift ignored any IDMS/ITMS created by the user and favored ICSP. With this release, they are allowed to co-exist as importing image stream tags will now respect IDMS/ITMS when ICSP is also present. (link:https://issues.redhat.com/browse/OCPBUGS-31469[*OCPBUGS-31469*])
```

Reply:
Not sure if this is the right way to go about it (my suggestion), but ITMS is not addressed specifically. Not sure if it needs to be?